src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner input = new Scanner(System.in);
+        int count = 0; //total number of tasks in the list - 1
+        Task[] Tasks = new Task[100];
+        Greet();
+        String userInput = input.nextLine();
+
+        while(!userInput.equals("bye")) {
+            //Print the list if input is 'list'
+            if (userInput.equals("list")) {
+                listTasks(count, Tasks);
+                userInput = input.nextLine();
+                continue;
+            }
+            //Mark the tasks as done
+            else if (userInput.startsWith("done")) {
+                markAsDone(Tasks, userInput);
+                userInput = input.nextLine();
+                continue;
+            }
+            else if (userInput.startsWith("deadline")) {
+                userInput = userInput.replace("deadline ", "");
+                String[] words = userInput.split(" /by ");
+                Tasks[count] = new Deadline(words[0], words[1]);
+                count++;
+            }
+            else if (userInput.startsWith("event")) {
+                userInput = userInput.replace("event ", "");
+                String[] words = userInput.split(" /at ");
+                Tasks[count] = new Event(words[0], words[1]);
```

Comment:
You are accessing a variable named Tasks of class Tasks, the similarity of the name might cause confusion.